client/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.css">
+    <link href="https://fonts.googleapis.com/css?family=Cardo|Montserrat" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="./style.css">
+    <title>My Travelling blog</title>
+</head>
+
+<body>
+    <div class="theme-switch-box">
+        <label class="theme-switch" for="switch">
+            <input type="checkbox" id="switch" />
+            <div class="slider round"></div>
+      </label>
+    </div>
+    <div id="root" class="main">
+
+
+    </div>
+
+    <div id="modal-box" class="modal__overlay">

```

Comment:
You could create the modal window in JS, too, and append it to the main container 😄